src/main/java/nextstep/calculator/domain/CalculatorElement.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.calculator.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CalculatorElement {
+
+    private static final String NEGATIVE_NUMBER_EXCEPTION_MESSAGE = "문자열 계산기에 음수는 포함될 수 없습니다.";
+    private List<Integer> elements;
```

Comment:
```suggestion
    private static final String NEGATIVE_NUMBER_EXCEPTION_MESSAGE = "문자열 계산기에 음수는 포함될 수 없습니다.";
    
    private List<Integer> elements;
```

클래스 변수와 인스턴스 변수 사이 공백을 추가한다면 가독성이 상승합니다 :)

Reply:
조언 감사합니다.